README.md

```diff
@@ -82,7 +82,7 @@ Links:
 
 Prior to the 1.26 release, enhancements from this repo were visualized using Enhancements Tracking Spreadsheets.
 
-Please refer to the [Enhancements Tracking Spreadsheet Archive](docs/archived-tracking-sheets.md) for links to 
+Please refer to the [Enhancements Tracking Spreadsheet Archive](docs/archived-tracking-sheets.md) for links to
 these sheets.
 
 Procedure:
```

Comment:
Was reading the documentation and thought it might be useful to update the 1.26 release information under the `Current Release Cycle` section

Reply:
Updated. Thanks.